### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by benbalter**
Realized we didn't have Rubocop for this project. Added Rubocop to clean up the code and disabled any tests that would require a larger refactor. This is largely the result of `rubocop -a`.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Metrics/LineLength:
+  Enabled: false
```
Comment (parkr) [inline .rubocop.yml]: If this is a real bummer, I'd recommend setting it to something slightly higher than reasonable like 120. We really shouldn't have code over 120 characters wide.

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
```
Comment (parkr) [inline .rubocop.yml]: If we're being strict, this is generally something we should try to deal with – maybe exclude the violating files for now and we can go through them later. Same with `Metrics/MethodLength`, `Metrics/CyclomaticComplexity`, `Metrics/PerceivedComplexity`, and `Metrics/AbcSize`.

File: github-pages-health-check.gemspec [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
-  s.name                  = "github-pages-health-check"
+  s.name                  = 'github-pages-health-check'
   s.version               = GitHubPages::HealthCheck::VERSION
-  s.summary               = "Checks your GitHub Pages site for commons DNS configuration issues"
-  s.description           = "Checks your GitHub Pages site for commons DNS configuration issues."
-  s.authors               = "GitHub, Inc."
-  s.email                 = "support@github.com"
-  s.homepage              = "https://github.com/github/github-pages-health-check"
-  s.license               = "MIT"
+  s.summary               = 'Checks your GitHub Pages site for commons DNS configuration issues'
```
Comment (parkr) [inline github-pages-health-check.gemspec]: I think it's our rule to always use double quotes, not single quotes. I'd borrow from the `github/pages-gem` rubocop.yml and make adjustments as necessary.

**Comment by benbalter** [comment]
@parkr ended up rewriting some of the tests to get the line lengths shorter. Please take another look.

**Pull request merged**